Autopilot/PathManager/Src/landingTakeoffManager.cpp

```diff
@@ -0,0 +1,202 @@
+#include "landingTakeoffManager.hpp"
+#include <math.h>
```

Comment:
If you defined PI in landingTakeoffManager.hpp, you can do:

```
#ifndef M_PI
#define M_PI 3.14159...
#endif
```

Reply:
what problem is this solving?